### PR TITLE
Add a section about managing chat notifications.

### PR DESCRIPTION
**Pull request by metropolislights**

### CODE REVIEW

File: source/docs/managing-chat-notifications.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src="/docs/assets/img/managing-chat-notifications/branch-whitelist.png" alt="Branch whitelist" class="img-responsive img-bordered">
+
+### Server checkboxes
+
+Untick any servers for which you do not wish to receive deploy notifications.
+
+<img src="/docs/assets/img/managing-chat-notifications/server-checkboxes.png" alt="Server checkboxes" class="img-responsive img-bordered">
+
+Save your settings once you're done, and you're done!
```
Comment (dunjardl) [inline source/docs/managing-chat-notifications.md]: Save your settings, and you're done!

File: source/docs/managing-chat-notifications.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+examples.
+
+<img src="/docs/assets/img/managing-chat-notifications/setting-up-notifications.png" alt="Setting up notifications" class="img-responsive img-bordered">
+
+Fill in the required form fields, click on **Save Settings**,  and you're all
+set to start receiving notifications!
+
+## Modifying notification policies
+
+For both builds and deploys, Semaphore offers several filters to better control
```
Comment (markoa) [inline source/docs/managing-chat-notifications.md]: This is not correct, for builds we have an additional option "Change status".

File: source/docs/managing-chat-notifications.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+what information you're subscribed to:
+
+- Always
```
Comment (markoa) [inline source/docs/managing-chat-notifications.md]: It'd be good to have a brief explanation for each here as well. For example:

- Always — send notifications after every build or deploy.
- Change status — (builds only) send notifications if build changes status from passed to failed and vice versa, and for all failed builds as well.

File: source/docs/managing-chat-notifications.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+For both builds and deploys, Semaphore offers several filters to better control
+what information you're subscribed to:
+
+- **Always** — Notify after every build or deploy.
+- **Never** — Do not notify after any build or deploy.
+- **Failed only** — Notify only if the build or deploy fails (or is stopped
+manually).
+- **Change status** (applies to builds only) — Notify only the build breaks or
+is fixed.
```
Comment (markoa) [inline source/docs/managing-chat-notifications.md]: breaks, continues to fail or has been fixed.

File: source/docs/managing-chat-notifications.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+### Choosing for which branches build notifications are sent
+
+Specifically for builds, you may choose to only receive notifications for a
+certain subset of branches. You can do this by selecting **From whitelist** in
+the **Select branches** dropdown box. A whitelist will appear, accepting [Ruby
+regular expressions](http://ruby-doc.org/core-2.2.3/Regexp.html). Each line
```
Comment (markoa) [inline source/docs/managing-chat-notifications.md]: I wouldn't say they're Ruby. AFAIK any standard regular expression will work.

Comment (metropolislights) [inline source/docs/managing-chat-notifications.md]: `issue-*` would match exactly that string, even though a non-Ruby regular expression would use the asterisk as a wildcard. Ruby regular expressions need `issue-.*`.

Comment (markoa) [inline source/docs/managing-chat-notifications.md]: ACK

**Pull request merged**